onnx/defs/tensor/utils.cc

```diff
@@ -88,6 +93,31 @@ void resizeShapeInference(InferenceContext& ctx) {
   const auto& input_shape = getInputShape(ctx, 0);
   auto* output_shape = getOutputShape(ctx, 0);
   const TensorProto* scales = 2 < ctx.getNumInputs() ? ctx.getInputData(2) : nullptr;
+  const TensorProto* sizes  = 3 < ctx.getNumInputs() ? ctx.getInputData(3) : nullptr;
+
+  if ((scales != nullptr) + (sizes != nullptr) != 1) {
```

Comment:
Unfortunately, this check is not the right one (for the warning message below). This condition checks whether a _constant-value_ is available. Unfortunately, there is no good predefined method/function to check whether an input is specified, but the following should work and be okay:
```cpp
   bool hasScalesInput = (2 < ctx.getNumInputs()) && (ctx.getInputType(2) != nullptr);
```